ftw/keywordwidget/configure.zcml

```diff
@@ -74,10 +114,13 @@
 
     <include package="ftw.upgrade" file="meta.zcml" />
     <upgrade-step:directory
+        zcml:condition="not-have plone-5"
```

Comment:
uhhh what's the reason for that? do we need different upgrade steps for plone 4 and plone 5?

Reply:
I removed it no. I added this in a desperate moment when registering wasn't working and forgot to remove it after that.